site/docs/voice/webCallingGuide.mdx

```diff
@@ -0,0 +1,202 @@
+---
+id: webCallingGuide
+title: Calling with WebRTC
+slug: /voice/guides/webrtc
+description: Using the WebRTC Connect product to place web-browser based calls into the Bandwidth voice network.
+keywords:
+  - bandwidth
+  - voice
+  - webrtc
+image: '@site/static/img/bw-icon.svg'
+---
+
+# Bandwidth WebRTC-Connect calling
+
+Bandwidth offers a *WebRTC Connect* calling capability that allows our customers to use WebRTC voice capabilities present in voice and mobile devices to place calls to critical destinations served by the Bandwidth voice network, allowing access to Bandwidth network services, quality and reach to advance any and all business needs surfaced by web browsers and mobile applications
```

Comment:
this line "Bandwidth offers a *WebRTC Connect* calling capability that allows our customers to use WebRTC voice capabilities present in voice" shall be "Bandwidth offers a *WebRTC Connect* calling capability that allows our customers to use WebRTC voice capabilities present in Browser"